force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,8 +104,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
-    }
+        if (a == null) {
+            return null;
+         } else if(Math.mod(a, 2) == 0) {
+            return 'Even'; 
+        } else {
+            return 'Odd'; 
+        } 
+        }
```

Comment:
Good work here. If you want to challenge yourself, solve it with a ternary operator. That is what I typically use for these sort of problems.